Dockerfile

```diff
@@ -3,8 +3,11 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal
 WORKDIR /usr/src/app
 
 ENV FLASK_RUN_HOST 0.0.0.0
+
 ENV BACKENDS_CONFIG_MAP=/etc/turnpike/backends.yml
-COPY ./Pipfile ./Pipfile.lock /usr/src/app/
```

Comment:
I vaguely remember the reason we only copy pipfile to there is: only if pipfile is updated we need to run the following install. Other files updates won't start again the install block.

Reply:
@astrozzc I am sorry I was faster, @petracihalova can you take a look ?

Reply:
no worries, it won't break anything :) @lpichler

Reply:
thank you @astrozzc, I will revert this